modules/creating-a-custom-ingress-controller.adoc

```diff
@@ -4,45 +4,54 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="creating-a-custom-ingress-controller_{context}"]
-= Creating a custom Ingress Controller with the `Unmanaged` DNS management policy
+= Creating an Ingress Controller for manual DNS management
 
-As a cluster administrator, you can create a new custom Ingress Controller with the `Unmanaged` DNS management policy.
+As a cluster administrator, you can create a new custom Ingress Controller with the Unmanaged DNS management policy.
 
 .Prerequisites
 
-* Install the OpenShift CLI (`oc`).
-* Log in as a user with `cluster-admin` privileges.
+* You have installed the OpenShift CLI (oc).
+* You are logged in as a user with cluster-admin privileges.
 
 .Procedure
 
-. Create a custom resource (CR) file named `sample-ingress.yaml` containing the following:
-
+. Create a custom resource (CR) file named sample-ingress.yaml with the following content:
 +
 [source,yaml]
 ----
 apiVersion: operator.openshift.io/v1
 kind: IngressController
 metadata:
   namespace: openshift-ingress-operator
-  name: <name> <1>
+  name: <name>
 spec:
-  domain: <domain> <2>
+  domain: <domain>
   endpointPublishingStrategy:
     type: LoadBalancerService
     loadBalancer:
-      scope: External <3>
-      dnsManagementPolicy: Unmanaged <4>
+      scope: External
+      dnsManagementPolicy: Unmanaged
+----
+
+. Apply the manifest to create the IngressController object:
++
+[source,terminal]
+----
+$ oc apply -f sample-ingress.yaml
 ----
-<1> Specify the `<name>` with a name for the `IngressController` object.
-<2> Specify the `domain` based on the DNS record that was created as a prerequisite.
-<3> Specify the `scope` as `External` to expose the load balancer externally.
-<4> `dnsManagementPolicy` indicates if the Ingress Controller is managing the lifecycle of the wildcard DNS record associated with the load balancer.
-The valid values are `Managed` and `Unmanaged`. The default value is `Managed`.
 
+. Verify that the Ingress Controller was created with the correct policy by running the following command:
++
+[source,terminal]
+----
+$ oc get ingresscontroller <name> -n openshift-ingress-operator -o yaml
+----
++
+Inspect the output and confirm that dnsManagementPolicy is set to `Unmanaged`, as shown in the example below: + [source,yaml]
 
 . Save the file to apply the changes.
 +
 [source,terminal]
 ----
-oc apply -f <name>.yaml <1>
+oc apply -f <name>.yaml
```

Comment:
What this step is doing exactly? The steps above applied the ingresscontroller manifest, so it's created and its initial definition is already in `sample-ingress.yaml`.